website/docs/en/guide/advanced/module-federation.mdx

```diff
@@ -28,7 +28,7 @@ import { PackageManagerTabs } from '@theme';
 
 <PackageManagerTabs command="add @module-federation/rsbuild-plugin -D" />
 
-Then add the plugin to the `rslib.config.ts` file:
+Then register the plugin to the `rslib.config.ts` file:
```

Comment:
```suggestion
Then register the plugin in `rslib.config.ts` file:
```

Reply:
Both "register the plugin to the rslib.config.ts file" and "register the plugin in rslib.config.ts file" are grammatically correct, but there's a subtle difference in meaning that might make one more appropriate depending on the context.
"Register the plugin to the rslib.config.ts file" implies a more direct action of linking the plugin to the file. It suggests the plugin is being added as a component or element within the file.
"Register the plugin in the rslib.config.ts file" implies a broader sense of inclusion within the file. It suggests the plugin is being recognized and acknowledged within the file's configuration.
Ultimately, the best choice depends on the specific way the plugin is being registered and the intended meaning.
If you're unsure, "register the plugin in the rslib.config.ts file" is a safer and more general option.